api/v1beta1/module_types.go

```diff
@@ -165,6 +165,11 @@ type ModuleLoaderContainerSpec struct {
 	// Sign provides default kmod signing settings
 	Sign *Sign `json:"sign,omitempty"`
 
+	// Version defines the current version of the kernel module being used
+	// Used for upgrading the currently loaded kernel module to a new version
+	// +kubebuilder:default="1.0"
```

Comment:
Why do we want a default value? I think we want this value to be completely optional (with the appropriate marker).

Reply:
in case user does not set the value of the version, we do want it to have value, even logically. User does not need to update it, if he does not want to use the ordered upgrade scenario. In addition, it is good to have a value in that field, since we will start differentiate DaemonSets of the ModuleLoader and DevicePlugins based not only on kernel mappings , but also on Version